gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,7 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
```

Comment:
Shall we use Gradle 8.11.1 here? We _might_ have to replace the wrapper JAR as well.

Reply:
done!